projects/desyn-farm1/index.js

```diff
@@ -2,7 +2,6 @@ const { getLogs } = require('../helper/cache/getLogs')
 
 module.exports = {
   methodology: 'On-chain restaking',
-  doublecounted: true
```

Comment:
if the tokens are not kept in the contract but moved outside, it should be marked as doublecounted since it's tvl is already counted towards chain tvl as part of the other protocol

Reply:
Okay, I'll re-add it doublecounted